src/Core/Services/OpenAPI/OpenApiDocumentor.cs

```diff
@@ -842,7 +843,7 @@ private static bool IsRequestBodyRequired(SourceDefinition sourceDef, bool consi
         private string GetEntityRestPath(string entityName)
         {
             string entityRestPath = entityName;
-            EntityRestOptions entityRestSettings = _runtimeConfig.Entities[entityName].Rest;
+            EntityRestOptions entityRestSettings = _runtimeConfigProvider.GetConfig().Entities[entityName].Rest;
```

Comment:
This is another location where you are mid-procedure and pulling a new version of the runtimeconfig. This presents opportunity for different sections of OpenAPI doc to contain contents of two different runtimeconfig objects.

NOTE: It may not be appropriate to pass the entire runtimeconfig object. Looking at this code, consider whether RuntimeConfig object even makes sense to use as argument:
- you may only need to pass in runtimeConfig.Entities[entityName].Rest -> in which case the function param should have input of `EntityRestOptions`